stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageContainer.kt

```diff
@@ -101,6 +103,9 @@ public fun MessageContainer(
             onGiphyActionClick = onGiphyActionClick,
             onMediaGalleryPreviewResult = onMediaGalleryPreviewResult,
             onQuotedMessageClick = onQuotedMessageClick,
+            onUserAvatarClick = {
+                onUserAvatarClick?.invoke(it.message.user.id)
```

Comment:
Instead of passing the userId, we should pass the `user` entity itself

Reply:
Nice have made that change - I did notice this was the pattern elsewhere in the code by force of habbit.